pkg/da/data_availability_header.go

```diff
@@ -193,13 +226,13 @@ func MinDataAvailabilityHeader() DataAvailabilityHeader {
 
 // MinShares returns one tail-padded share.
 func MinShares() [][]byte {
-	return share.ToBytes(EmptySquareShares())
+	return sharev3.ToBytes(EmptySquareShares())
 }
 
 // EmptySquareShares is a copy of the function defined in the square package to avoid
 // a circular dependency. TODO deduplicate
-func EmptySquareShares() []share.Share {
-	return share.TailPaddingShares(appconsts.MinShareCount)
+func EmptySquareShares() []sharev3.Share {
```

Comment:
To be sure: there's no difference between squarev3 empty square shares and squarev2 empty square shares?

Reply:
Ah I guess that's what the backwards compat test is for 👍🏻  all g